src/util.ts

```diff
@@ -61,3 +61,9 @@ export const endReporter = (
     }
   }
 };
+
+export const unmockUAHeaderValue = () => {
+  return { lang: isNode ? "node" : "jsdom" };
```

Comment:
Like you said - could be more specific. Even proliferation with e.g. `{ lang: "js", framework: isNode ? "node" : "jsdom", version: require("../package.json").version }` might be a good start (with `lang` reserved to the unmock client being used).

Reply:
I'll add the version, about the "lang" + framework I'm not that sure as it would just make analytics harder if one needs to split by both lang and framework? And framework may not make sense for other languages so wouldn't that be an extra complication?

Reply:
Hm, that's true. Guess either one would be good enough for now :grinning:

Reply:
For reference, here's what Stripe does for Node: https://github.com/stripe/stripe-node/blob/master/lib/stripe.js#L13 and https://github.com/stripe/stripe-node/blob/master/lib/stripe.js#L294

Reply:
Sure, has plenty of options. Their Python SDK adds a few more keys - https://github.com/stripe/stripe-python/blob/master/stripe/api_requestor.py#L238

Reply:
I'll just merge this now and ask Mike if there's a way to add the version that works in browser as well.